website/docs/tests/what-is-a-test.md

```diff
@@ -86,11 +98,52 @@ Assert:
 
 The error operation lets you define a set of expected errors for a test step. If any of these errors occur during the test, they are treated as expected outcomes. However, if an error that's not on this list occurs, it will be treated as a test failure.
 
+#### Fields Description
+
+| Field          | Type      | Notes                                                                                       |
+|----------------|-----------|---------------------------------------------------------------------------------------------|
+| FileRef        | FileRef   | Provides a reference to the file containing the expected error.                             |
+| ContinueOnError| Bool      | Optional. Determines if the test should continue when the operation isn't successful.       |
+
 ```yaml
 Error:
   - file: path/to/expected-errors.yaml
 ```
 
+### Exec operations {#exec}
+
+The `Exec` operation provides a means to either execute a specific command or run a script during the test step. For each `Exec` entry, you must specify either a `Command` or a `Script`, but not both.
+
+#### Fields Description
+
+| Field    | Type                | Notes                                                              |
+|----------|---------------------|--------------------------------------------------------------------|
+| Timeout  | `metav1.Duration`   | Optional. Overrides the global timeout set in the Configuration.   |
+| Command  | `Command`           | Optional. Defines a specific command to be run.                    |
+| Script   | `Script`            | Optional. Defines a specific script to be executed.                |
```

Comment:
The exec operation has more fields than this, i would rather point to the API reference docs instead of repeating it here.